### PR TITLE
Update configure-data-collection.md

### PR DESCRIPTION
**Pull request by pbrzoska-sumo**
## Purpose of this pull request

- minor corrections in intro and prerequisites
- added info about no geolocations when using intermediary collector

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/apm/real-user-monitoring/configure-data-collection.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import Iframe from 'react-iframe';
 
-To collect [traces](/docs/apm/traces) from a browser, you'll first need to create a RUM HTTP Traces Source. The source will have an endpoint URL that you'll put in a script that sends trace data in [OTLP/JSON over HTTP](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/protocol/otlp.md#otlphttp) protocol.
+To collect [traces](/docs/apm/traces) and RUM mertrics from a browser, you'll first need to create a RUM HTTP Traces Source. The source will have an endpoint URL that you'll put in a script that sends trace data in [OTLP/JSON over HTTP](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/protocol/otlp.md#otlphttp) protocol. Alternatively you can also use an intermediary OTel collector, if you require data to flow over your infrastructure rather than directly to Sumo Logic. Note however this will disable automatic geo-location recognition capabilities.
```
Comment (JainM6) [inline docs/apm/real-user-monitoring/configure-data-collection.md:11]: I'd rephrase it as:
The source will have an endpoint URL that you can use in a script to send [OTLP/JSON over HTTP](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/protocol/otlp.md#otlphttp) protocol trace data.

Comment (JainM6) [inline docs/apm/real-user-monitoring/configure-data-collection.md:11]:
```suggestion
To collect [traces](/docs/apm/traces) and RUM metrics from a browser, you'll first need to create a RUM HTTP Traces Source. The source will have an endpoint URL that you can use in a script to send [OTLP/JSON over HTTP](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/protocol/otlp.md#otlphttp) protocol trace data. Alternatively, you can also use an intermediary OTel collector if you require data to flow over your infrastructure rather than directly to Sumo Logic. Note however this will disable automatic geo-location recognition capabilities.
```

File: docs/apm/real-user-monitoring/configure-data-collection.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ See Real User Monitoring in action.
 :::
 
 ## Prerequisites
-XHR and navigation/route changes support as well as errors collection require RUM script in version 4 or higher (`https://rum.sumologic.com/sumologic-rum-v4.js`). Make sure you're using the correct version in your pages. For automatic updates, use `https://rum.sumologic.com/sumologic-rum.js`.
+XHR and navigation/route changes support as well as errors collection require RUM script in version 4 or higher (`https://rum.sumologic.com/sumologic-rum-v4.js`). Make sure you're using the correct version in your pages. For automatic updates, use `https://rum.sumologic.com/sumologic-rum.js`. You can find more details about versioning control later in this document.
```
Comment (JainM6) [inline docs/apm/real-user-monitoring/configure-data-collection.md:30]:
```suggestion
Support for XHR and navigation/route changes, as well as error collection, require RUM script version 4 or higher ('https://rum.sumologic.com/sumologic-rum-v4.js'). Make sure you're using the correct version in your pages. For automatic updates, use `https://rum.sumologic.com/sumologic-rum.js`. You can find more details about versioning control later in this document.
```
```suggestion
XHR and navigation/route changes support, and errors collection require the RUM script in version 4 or higher (`https://rum.sumologic.com/sumologic-rum-v4.js`). Make sure you're using the correct version on your pages. For automatic updates, use `https://rum.sumologic.com/sumologic-rum.js`. You can find more details about versioning control later in this document.
```

**Review by JainM6 — CHANGES_REQUESTED** [review]
Formatting changes

File: docs/apm/real-user-monitoring/configure-data-collection.md (hunk 1 above)
Comment (kimsauce) [inline docs/apm/real-user-monitoring/configure-data-collection.md:11]:
```suggestion
To collect [traces](/docs/apm/traces) and RUM metrics from a browser, you'll first need to create a RUM HTTP Traces Source. The source will have an endpoint URL that you'll put in a script that sends trace data in [OTLP/JSON over HTTP](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/protocol/otlp.md#otlphttp) protocol. Alternatively, you can also use an intermediary OTel collector, if you require data to flow over your infrastructure rather than directly to Sumo Logic. Note however this will disable automatic geo-location recognition capabilities.
```

**Review by kimsauce — APPROVED** [review]

**Review by stacykor — APPROVED** [review]
added RUM metrics and a note.

File: docs/apm/real-user-monitoring/configure-data-collection.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ See Real User Monitoring in action.
 :::
 
 ## Prerequisites
-XHR and navigation/route changes support as well as errors collection require RUM script in version 4 or higher (`https://rum.sumologic.com/sumologic-rum-v4.js`). Make sure you're using the correct version in your pages. For automatic updates, use `https://rum.sumologic.com/sumologic-rum.js`.
+XHR and navigation/route changes support, and errors collection require the RUM script in version 4 or higher. To know more, refer to the link `https://rum.sumologic.com/sumologic-rum-v4.js`. Make sure you're using the correct version on your pages. For automatic updates, use `https://rum.sumologic.com/sumologic-rum.js`. You can find more details about versioning control later in this document.
```
Comment (kimsauce) [inline docs/apm/real-user-monitoring/configure-data-collection.md:30]:
```suggestion
To utilize XHR and navigation/route changes, and errors collection, you must use RUM script version 4 or higher (`https://rum.sumologic.com/sumologic-rum-v4.js`). Make sure you're using the correct version in your pages. For automatic updates, use the script `https://rum.sumologic.com/sumologic-rum.js`. You can find more details about versioning control [later in this document](#step-2-add-rum-script-to-your-page-header).
```
